docker/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM firedrakeproject/firedrake:2024-05 AS spyro_base
+
+USER root
+RUN apt-get update \
+    && apt-get -y dist-upgrade \
+    && apt-get -y install gmsh patchelf \
+    && rm -rf /var/lib/apt/lists/*
+
+USER firedrake
+RUN . /home/firedrake/firedrake/bin/activate; pip3 install wheel --upgrade
+RUN . /home/firedrake/firedrake/bin/activate; pip3 install scipy
+RUN . /home/firedrake/firedrake/bin/activate; pip3 install roltrilinos
+RUN . /home/firedrake/firedrake/bin/activate; pip3 install ROL
+RUN . /home/firedrake/firedrake/bin/activate; pip3 install matplotlib
+RUN . /home/firedrake/firedrake/bin/activate; pip3 install segyio
+
+# spyro dependencies
+USER root
+# Is there a reason for using development versions?
```

Comment:
I am not sure. I haven't tried with other versions. Line 20 comes from SeismicMesh dependencies and the first iteration of its installation tutorial. I guess that only the dev version of Cgal is needed, and that is only for compatibility purposes in older Ubuntu installations.

Reply:
Ok. I will remove this comment